packages/react-composites/scripts/runBrowserTests.mjs

```diff
@@ -61,14 +63,34 @@ async function runStress(args) {
 }
 
 async function runAll(args) {
+  let success = true;
   for (const composite of args.composites) {
-    await runOne(args, composite, 'hermetic');
+    try {
```

Comment:
I realize this was in before but we are saying we always run all of the hermetic tests and then if we set !hermeticOnly we run the live tests?

is there no chance we would ever just want to run live and not hermetic?

Reply:
At least right now, I didn't see a need for this. We can definitely tweak that in the future if we want.

Reply:
Note that you can always run a specific test / test file etc by passing arguments to playwright (after `--`)